### PR TITLE
Sign in new layout

### PR DESCRIPTION
**Pull request by LaviniaDragunoi**
Added XML files for Login feature.

### CODE REVIEW

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+android:background="@color/white">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (Elforama) [inline app/src/main/res/layout/fragment_login.xml]: Why is the ConstraintLayout nested in a FrameLayout? Can we remove the frame layout?

File: app/src/main/res/layout/fragment_sign_up.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (Elforama) [inline app/src/main/res/layout/fragment_sign_up.xml]: Same as above

File: app/src/main/res/layout/fragment_terms_and_policy.xml [hunk 3]
```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="@dimen/terms_text_body">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (Elforama) [inline app/src/main/res/layout/fragment_terms_and_policy.xml]: Same as above

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -150,6 +165,24 @@
     <string name="identification_text">Show us who you are\.\.\.</string>
     <string name="start_planting">Start Planting</string>
 
-    <string name="build_version_title" translatable="false">Build version  </string>
-    <string name="tree_tracker_title" translatable="false">Tree Tracker(debug) </string>
+    <string name="build_version_title" translatable="false">Build version</string>
+    <string name="tree_tracker_title" translatable="false">Tree Tracker(debug)</string>
+
+    <!--SignUp screen strings-->
+    <string name="greenstand_welcome_text" translatable="false">Greenstand Tree Tracker</string>
```
Comment (Elforama) [inline app/src/main/res/values/strings.xml]: Is the signup screen only supposed to be in english? I see `translatable = false` on all strings

**Comment by LaviniaDragunoi** [comment]
I will start working for the new design in another branch having all last updated version of the app.

**Comment by deepwinter** [comment]
@LaviniaDragunoi What is the status of this branch vs the Login Feature branch ?

**Comment by LaviniaDragunoi** [comment]
@deepwinter I will remove this PR because I am working on new_login_feat branch for this.
 Hope this is ok with you.

**Pull request closed without merging**